scripts/common.py

```diff
@@ -666,7 +674,9 @@ def inplace_replace_file(
 
 
 @contextlib.contextmanager
-def inplace_replace_file_context(filename, regex_pattern, replacement_string):
+def inplace_replace_file_context(
+    filename: str, regex_pattern: str, replacement_string: str
+) -> Iterator[None]:
```

Comment:
Are you sure this is an `Iterator`? Isn't there a different type for context?

Reply:
Sorry, I am not sure about which type you are talking about. Can you please mention that? And after reading [this](https://stackoverflow.com/questions/49733699/python-type-hints-and-context-managers), I think we can use Iterator or Generator. Please correct me if I am wrong and I would like to have your suggestion. Thanks,